doc/game.md

```diff
@@ -49,27 +55,38 @@ To remove components from the list on a `BaseGame` the `markToRemove` method can
 
 ## Flutter Widgets and Game instances
 
-Since a Flame game is a widget itself, it is quite easy to use Flutter widgets and Flame game together. But to make it even easier, Flame provides a `mixin` called `HasWidgetsOverlay` which will enable any Flutter widget to be show on top of your game instance, this makes it very easy to create things like a pause menu, or an inventory screen for example.
+Since a Flame game can be wrapped in a widget, it is quite easy to use it alongside other Flutter widgets. But still, there is a the Widgets overlay API that makes things even easier.
+
+Flame provides a `mixin` called `HasWidgetsOverlay` which will enable any Flutter widget to be shown on top of your game instance, this makes it very easy to create things like a pause menu, or an inventory screen for example.
+
+To use it, simple add the `HasWidgetsOverlay` `mixin` on your game class, by doing so, the game class will have a `.overlays` property that will be used to manage the active overlays.  
+This management happens via the `.overlays.add` and `.overlays.remove` methods that marks a overlay to be shown and hidden, respectively, via a `String` argument that identifies a overlay.  
```

Comment:
First occurrence: a overlay -> an overlay
Second occurrence: a overlay -> the overlay